tests/molecules/C4_ae/CMakeLists.txt

```diff
@@ -0,0 +1,150 @@
+
+IF (NOT QMC_CUDA)
+ IF (NOT QMC_COMPLEX)
+
+# E_scf_QP=-150.95914112726956
+#  Reference from VMC runs
+  LIST(APPEND C4_SD_AE_QP_SHORT "totenergy" "-150.960566064824 0.074761412109")
```

Comment:
This should use E_scf above, right?

Reply:
If not, write an explicit comment in text justifying the difference in reference for SCF and VMC.

Reply:
Done